constants.js

```diff
@@ -1,10 +1,27 @@
+// Use .env for dev, but use process.env directly in production (eg, for Heroku)
+if (process.env.NODE_ENV !== "production" && process.env.NODE_ENV !== "test") {
+  const dotenvConfig = require("dotenv").config(); 
+  
+  if (dotenvConfig.error) {
+    throw dotenvConfig.error;
+  }
+}
+
+const port = process.env.PORT || 3000;
 const sentryAPIbase = process.env.SENTRY_API_BASE || "https://sentry.io/api/0";
+const sentryAPIToken = process.env.SENTRY_TOKEN;
```

Comment:
@LearningNerd So these aren't being mocked in tests? I'm a bit concerned that the tests pass when all of these values are `undefined`.

Reply:
Oh these are being mocked for the tests via this file that runs before any of the test suites run: https://github.com/getsentry/sentry-round-robin/blob/master/__tests__/jest.setup.js

Reply:
@LearningNerd I see. I think we should avoid mutating `process.env` in tests like that. Better off mutating the values of the module export.

Reply:
Yup you're right, much better that way! I updated it. 👍 Thanks!